superset/assets/stylesheets/superset.less

```diff
@@ -535,3 +535,6 @@ tr.reactable-column-header th.reactable-header-sortable {
 .align-right {
   text-align: right;
 }
+.margin-zero {
+  margin: 0px !important;
```

Comment:
Don't think we need the `!important` here.

Reply:
I added it because it was needed...

Reply:
could be change to `img.loading.margin-zero` to avoid using `!important`

Reply:
happy to change it, I usually start using interpolation when there's more than one `+`, would be nice to get the linter to trigger on this if you all feel like we should use interpolation more systematically.

Reply:
The goal was to make a reusable class that kind de-margin anything, that's why I put it in `superset.less`, as a targeted class I think it belongs in `Loading.css`...